src/main/java/dansplugins/factionsystem/integrators/CurrenciesIntegrator.java

```diff
@@ -12,6 +12,6 @@
 public class CurrenciesIntegrator {
 
     public boolean isCurrenciesPresent() {
-        return (Bukkit.getServer().getPluginManager().getPlugin("Currencies") != null);
+        return Bukkit.getServer().getPluginManager().getPlugin("Currencies") == null;
```

Comment:
This inverts the logic! Surely if we want to check currencies _is_ present this should be returning that it isn't null?

Reply:
It will return false if plugin installed

Reply:
That runs counter to the method's name: `isCurrenciesPresent` should return whether currencies is present, not missing.
Either the change in behaviour should be reverted or the method should be renamed to make the intent clear.

Reply:
oki, i'll change